cmd/airgap/listimages.go

```diff
@@ -34,19 +33,23 @@ func NewAirgapListImagesCmd() *cobra.Command {
 		Example: `k0s airgap list-images`,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			// we don't need warning messages in case of default config
-			logrus.SetLevel(logrus.ErrorLevel)
-			c := CmdOpts(config.GetCmdOpts())
-			cfg, err := config.GetYamlFromFile(c.CfgFile, c.K0sVars)
+			// logrus.SetLevel(logrus.ErrorLevel)
```

Comment:
I think these type of cases the logrus output could be sent to stderr so that piping would not include the logs.

Reply:
commented code is leftover